src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/integration/PrisonOffenderEventListenerTest.kt

```diff
@@ -101,11 +100,8 @@ class PrisonOffenderEventListenerTest : SqsIntegrationTestBase() {
     )
 
     awaitAtMost30Secs untilAsserted {
-      verify(done).complete()
+      verifyNoInteractions(telemetryClient)
```

Comment:
mmh.. wonder if this is passing with false positives, not sure you can assert a negative - this would pass if no message was raised

Reply:
true! Kind of existing code